amazon/ionbenchmark/ion_load_dump.py

```diff
@@ -26,12 +26,17 @@ def loads(self, s):
 
     def load(self, fp):
         ion.c_ext = self._c_ext
-        return ion.load(fp, single_value=self._single_value, value_model=self.value_model)
+        it = ion.load(fp, parse_eagerly=False, single_value=False, value_model=self.value_model)
+        while True:
+            try:
+                yield next(it)
+            except StopIteration:
+                return
 
     def dumps(self, obj):
         ion.c_ext = self._c_ext
-        return ion.dumps(obj, binary=self._binary)
+        ion.dumps(obj, binary=self._binary)
```

Comment:
Should this use `sequence_as_stream=True`?

Reply:
io_type=buffer currently doesn't handle multiple top-level use case, I opened an issue for that - https://github.com/amazon-ion/ion-python/issues/325